org.adempiere.base.callout/src/org/compiere/model/CalloutInventory.java

```diff
@@ -173,5 +173,50 @@ private BigDecimal setQtyBook (int M_AttributeSetInstance_ID, int M_Product_ID,
 		return Env.ZERO;
 	}
 	
-
+	/**
+	 * 
+	 * 	reset value of org when not set charge 
+	 * 
+	 *  @param ctx      Context
+	 *  @param WindowNo current Window No
+	 *  @param mTab     Model Tab
+	 *  @param mField   Model Field
+	 *  @param value    The new value
+	 *  @return Error message or ""
+	 */
+	public String setCharge (Properties ctx, int WindowNo, GridTab mTab, GridField mField, Object value){		

```

Comment:
the names of the methods here are misleading
- setCharge is not setting a charge - maybe a correct name would be setOrgFromParent
- samely setInventoryType - as both have the same purpose setting the Org from parent maybe they can be the same method and discover from which field is called